ivadomed/config_manager.py

```diff
@@ -0,0 +1,51 @@
+import json
+import os
+
+__ivadomed_dir__ = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+
+
+def _load_json(config_path):
+    """Load json file content
+
+    Args:
+        config_path (str): Path to json file.
+
+    """
+    with open(config_path, "r") as fhandle:
+        default_config = json.load(fhandle)
+    return default_config
+
+
+class ConfigurationManager(object):
+    """Configuration file manager
+
+    Args:
+        context_path (str): Path to configuration file.
+
+    Attributes:
+        context_path (str): Path to configuration file.
+        default_config (str): Path to default configuratio file.
+
+    """
+    def __init__(self, context_path):
+        self.context_path = context_path
+        self._validate_path()
+        default_config_path = os.path.join(__ivadomed_dir__, "ivadomed", "config", "config_default.json")
+        self.default_config = _load_json(default_config_path)
+        self.context = _load_json(context_path)
+
+    def get_config(self):
+        """Get updated configuration file with all parameters from the default config file.
+
+        Returns:
+            dict: Updated configuration dict.
+        """
+        self.default_config.update(self.context)
+        return self.default_config
```

Comment:
Great!
What about adding something like: if debbugging is True, then display the keys that differ between the default and the context?

Reply:
Updated here: 7e42580